Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
+
+	}
+}
+
+// Should tell me anonymous class are not available in PHP 5.6
+$a = new class {}
+
+// Should tell me "create_function()" has been deprecated on PHP 7.2
+create_function('', '');
\ No newline at end of file
```

Comment:
* :warning: create_function() is highly discouraged, as it can execute arbritary code (additionally, it's deprecated as of PHP 7.2): https://wpvip.com/documentation/vip-go/code-review-blockers-warnings-notices/#eval-and-create_function. )
* :warning: Function create_function() is deprecated since PHP 7.2; Use an anonymous function instead
* :no_entry_sign: PHP syntax error: syntax error, unexpected 'create_function' (T_STRING)
* :no_entry_sign: create_function() is deprecated as of PHP 7.2, please use full fledged functions or anonymous functions instead.
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: Function create_function() has been deprecated
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: File must end with a newline character

<sub>hash: 5768da684300b0e405a37232e76f4b56</sub>